tests/library/trace-viewer.spec.ts

```diff
@@ -676,7 +676,7 @@ test('should open two trace files', async ({ context, page, request, server, sho
 
 test('should include requestUrl in route.fulfill', async ({ page, runAndTrace, browserName }) => {
   await page.route('**/*', route => {
-    route.fulfill({
+    void route.fulfill({
```

Comment:
I don't quite like this `void` prefixes all over the place.

Reply:
notes:
+: don't change existing tests -> no logic changes
-: have something different to what we recommend to our users (+ we have async route handler tests already)